docs/dev-guide/migration-flowchart.d2

```diff
@@ -0,0 +1,69 @@
+direction: down
+vars: {
+  d2-config: {
+    layout-engine: elk
+  }
+}
+
+with_default: "Column with default value?" {
+  shape: diamond
+  style.fill: "#e8f5e8"
+}
+
+with_constraint: "Do you need to enforce constraints\n (e.g. NOT NULL, CHECK)?" {
+  shape: diamond
+  style.fill: "#fff3e0"
+}
+
+simple_migration: "Single migration" {
+  style.fill: "#c8e6c9"
+}
+
+pr1: "PR 1" {
+  style.fill: "#e3f2fd"
+  explanation: |md
+
+    Two migrations:
+
+    - add column
+    - enqueue backfill using a batched migration
+
+    .
```

Comment:
For some reason, text was getting cutoff when the explanation field ended in a list, hence the `.`.